torchvision/transforms/functional_pil.py

```diff
@@ -261,7 +290,13 @@ def _parse_fill(fill, img, name="fillcolor"):
 
 
 @torch.jit.unused
-def affine(img, matrix, interpolation=0, fill=None):
+def affine(
+    img: Image.Image,
+    matrix: List[float],
+    interpolation: int = 0,
```

Comment:
Perhaps we should us Image.NEAREST instead of the numeric here?

Reply:
Hmm, something stranger is that in the tensor API we pass a string.
I'm not sure how would we align it. (again probably JIT?)

Reply:
I mean something different. See the next comment.